From c94e669de98a3892c699bd8d0d2b5164b2de747e Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 15 Mar 2014 22:42:29 -0700
Subject: [PATCH] Fix DATADIRNAME on uclibc/Linux

translation files are always installed under PREFIX/share/locale in uclibc
based systems therefore lets set DATADIRNAME to "share".

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Upstream-Status: Pending


---
 m4macros/glib-gettext.m4 | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/m4macros/glib-gettext.m4 b/m4macros/glib-gettext.m4
index df6fbf0..47db864 100644
--- a/m4macros/glib-gettext.m4
+++ b/m4macros/glib-gettext.m4
@@ -293,6 +293,10 @@ msgstr ""
 	    CATOBJEXT=.mo
             DATADIRNAME=share
 	    ;;
+	    *-*-musl* | *-*-linux-uclibc*)
+	    CATOBJEXT=.gmo
+            DATADIRNAME=share
+	    ;;
 	    *)
 	    CATOBJEXT=.mo
             DATADIRNAME=lib
